config/nginx/prod/nginx.conf

```diff
@@ -156,7 +156,7 @@ server {
     # - `/quotes/`
     # - any path starting with `/api/auth/` or `/auth/`
     # To add a route, insert `|your_path/?$` before the closing parenthesis.
-location ~ ^/($|_?_next.*|search/?$|entities/?$|occurrences/.*|topics/?$|quotes/.*|api/auth/.*|auth/.*|about/?$|terms/?$|privacy/?$|manifesto/?$|board-of-directors/?$) {
+    location ~ ^/($|_?_next.*|search/?$|entities/?$|occurrences.*|topics/?$|quotes.*|api/auth/.*|auth/.*|about/?$|terms/?$|privacy/?$|manifesto/?$|board-of-directors/?$) {
```

Comment:
Since `donations` is not in this location directive, we won't be able to go to the donations page in production yet.  But I think that's fine, since the donations page is currently only hooked up to our Braintree sandbox account and is not ready to process actual donations. Once we're ready for the donations page to be accessed in production, we can add it to prod/nginx.conf.